src/main/java/com/bankingservice/banking/exception/InsertionFailedException.java

```diff
@@ -0,0 +1,41 @@
+package com.bankingservice.banking.exception;
+
+public class InsertionFailedException extends Throwable {
```

Comment:
The custom exception should be child of Exception Class or Runtime Exception class, do R&D on this, you will get the scenarios when to use which case.